bluesky/callbacks/core.py

```diff
@@ -686,13 +686,11 @@ def descriptor(self, doc):
         """Write the header for the actual scan data"""
         # List all scalar fields, excluding the motor (x variable).
         self._read_fields = sorted([k for k, v in doc['data_keys'].items()
-                                    if (v['object_name'] != self._motor
```

Comment:
Before I can fully respond, I need some information. Where do the names in `start_document['motors']` come from?